package.json

```diff
@@ -33,6 +33,13 @@
       "default": true,
       "order": 90
     },
+    "findReferencesHyperclick": {
```

Comment:
Since hyperclick on the definition isn't all that useful, I think we may drop the config option and instead enable this unconditionally.

Reply:
I agree that and I did drop that option. Thanks!